ui/webui/src/components/users/Accounts.jsx

```diff
@@ -55,6 +58,24 @@ export const accountsToDbusUsers = (accounts) => {
     }];
 };
 
+const isUserAccountReserved = (userAccount) => {
+    return (
+        userAccount === "root" ||
+        userAccount === "daemon" ||
+        userAccount === "home" ||
+        userAccount === "system"
```

Comment:
I could image this list in our config files, but it is probably not necessary.

Reply:
Yes, or it will rather go to the backend with the validation.
Actually I was using wrong branch for the list so I had to extend it a bit and rebase the PR (sorry).